.spelling

```diff
@@ -180,6 +180,8 @@ SendPing
 SERP
 serverless
 Sonatype
+spark-bigquery
```

Comment:
If you put `spotify/spark-bigquery` in backticks in the main text (per suggested change), they don't need to go into the `.spelling` file.
```suggestion
```